cmd/kubectl-gadget/utils/flags.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type CommonFlags struct {
+	Label         string
+	Node          string
+	Podname       string
```

Comment:
Jose suggested to add a comment to explain why we don't have a `Namespace` field.